contracts/stablecoinswap.vy

```diff
@@ -103,9 +103,9 @@ def removeLiquidity(token_address: address, stableswap_token_amount: uint256, er
 
     token_price: uint256 = self.tokenPrice(token_address)
     pool_size: uint256 = PriceOracle(self.priceOracleAddress).poolSize(self)
-    # erc20_token_amount = stableswapt_token_amount * pool_size / totalSupply / token_price * TOKEN_PRICE_MULTIPLIER
+    # erc20_token_amount = stableswap_token_amount * pool_size / totalSupply / token_price * TOKEN_PRICE_MULTIPLIER
     # It's better to divide at the very end for a higher precision
-    erc20_token_amount: uint256 = stableswap_token_amount * pool_size * TOKEN_PRICE_MULTIPLIER / token_price / self.totalSupply
+    erc20_token_amount: uint256 = stableswap_token_amount * pool_size / self.totalSupply
```

Comment:
I've moved conversion from stableswap tokens to output tokens below for a higher precision. During testing I became `erc20_token_amount` as `2003.996`. It was truncated to `2003`. After swap fee was substracted I got `2003 * 0.997 = 1996`. It is lower than expected amount: `2003.996 * 0.997 = 1997`.